wasm.md

```diff
@@ -689,12 +706,22 @@ Type could be declared explicitly and could optionally bind with an identifier.
                       | "(type" Identifier "(" "func" TypeDecls ")" ")"
  // -------------------------------------------------------------------
     rule <k> (type (func TDECLS:TypeDecls)) => . ... </k>
-         <nextTypeIdx> NEXTIDX => NEXTIDX +Int 1 </nextTypeIdx>
-         <types>       TYPES   => TYPES [NEXTIDX <- asFuncType(TDECLS)]   </types>
+         <curModIdx> CUR </curModIdx>
+         <moduleInst>
+           <modIdx> CUR </modIdx>
+           <nextTypeIdx> NEXTIDX => NEXTIDX +Int 1 </nextTypeIdx>
+           <types>       TYPES   => TYPES [NEXTIDX <- asFuncType(TDECLS)]   </types>
```

Comment:
spacing here is a bit weird, can we remove the extra spaces before `</types>` or something?